bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -35,226 +35,230 @@ Location,Garden,Outdoor,Garden,Gardens,
-Property,Presence,,Presence,,
+Property,Presence,,Presence,,Presence detection
 Property,Pressure,,Pressure,,
-Property,QualityOfService,,Quality of Service,,
-Property,Rain,,Rain,,
+Property,Progress,,Progress,,
+Property,QualityOfService,,Quality of Service,,Quality of service
+Property,Rain,,Rain,,Rain measurement
 Property,SignalStrength,,Signal Strength,,
-Property,RSSI,SignalStrength,RSSI,Received Signal Strength Indication,
-Property,Smoke,,Smoke,,
-Property,SoundVolume,,Sound Volume,,
-Property,Speed,,Speed,,
-Property,Tampered,,Tampered,,
-Property,Temperature,,Temperature,Temperatures,
-Property,Tilt,,Tilt,,
-Property,Timestamp,,Timestamp,,
-Property,Ultraviolet,,Ultraviolet,UV,
+Property,RSSI,SignalStrength,RSSI,Received Signal Strength Indication,Received signal strength indication
+Property,Smoke,,Smoke,,Smoke detection
+Property,SoundVolume,,Sound Volume,,Sound volume level
+Property,Speed,,Speed,,Speed measurement or setting
+Property,Tampered,,Tampered,,Tampering detected
+Property,Temperature,,Temperature,Temperatures,Temperature measurement
+Property,Tilt,,Tilt,,"Tilt of a blind, shade or shutter"
+Property,Timestamp,,Timestamp,,Timestamp of an event
+Property,Ultraviolet,,Ultraviolet,UV,Ultraviolet measurement
 Property,Ventilation,,Ventilation,,
-Property,Vibration,,Vibration,,
-Property,Voltage,,Voltage,,
-Property,Water,,Water,,
-Property,Wind,,Wind,,
-Equipment,AlarmDevice,,Alarm Device,,
-Equipment,AlarmSystem,,Alarm System,Alarm Systems,
-Equipment,AudioVisual,,Audio Visual,,
-Equipment,Display,AudioVisual,Display,,
-Equipment,Projector,Display,Projector,"Projectors, Beamer, Beamers",
-Equipment,Television,Display,Television,"Televisions, TV, TVs",
-Equipment,MediaPlayer,AudioVisual,Media Player,,
-Equipment,Receiver,AudioVisual,Receiver,"Receivers, Audio Receiver, Audio Receivers, AV Receiver, AV Receivers",
-Equipment,Screen,AudioVisual,Screen,Screens,
-Equipment,Speaker,AudioVisual,Speaker,Speakers,
-Equipment,Bed,,Bed,,
-Equipment,Camera,,Camera,Cameras,
-Equipment,CleaningRobot,,Cleaning Robot,"Cleaning Robots, Vacuum robot, Vacuum robots",
-Equipment,Computer,,Computer,,
-Equipment,ControlDevice,,Control Device,,
-Equipment,Button,ControlDevice,Button,,
-Equipment,Dial,ControlDevice,Dial,Rotary Dial,
-Equipment,Keypad,ControlDevice,Keypad,,
-Equipment,Slider,ControlDevice,Slider,,
-Equipment,WallSwitch,ControlDevice,Wall Switch,Wall Switches,
-Equipment,Door,,Door,Doors,
-Equipment,BackDoor,Door,Back Door,Back Doors,
-Equipment,CellarDoor,Door,Cellar Door,Cellar Doors,
-Equipment,FrontDoor,Door,Front Door,"Front Doors, Frontdoor, Frontdoors",
-Equipment,GarageDoor,Door,Garage Door,Garage Doors,
-Equipment,Gate,Door,Gate,Gates,
-Equipment,InnerDoor,Door,Inner Door,Inner Doors,
-Equipment,SideDoor,Door,Side Door,Side Doors,
-Equipment,Doorbell,,Doorbell,Doorbells,
+Property,Vibration,,Vibration,,Vibration level
+Property,Voltage,,Voltage,,Electric voltage
+Property,Water,,Water,,Water measurement
```

Comment:
Can this also be used as a property of Control (to control water flow)? Therefore probably don’t include measurement in the description.